package/tvheadend-app-osmc/files/DEBIAN/postinst

```diff
@@ -41,6 +94,61 @@ if [ "$1" = "configure" ]; then
 		fi
 	fi
 
-	if ischroot; then exit 0; fi
+	# start tvh so that it updates any old user files
+	echo Starting tvheadend to finalise setup for osmc
 	if systemctl is-enabled tvheadend.service >/dev/null; then systemctl start tvheadend.service; fi
+	#systemctl restart tvheadend
+	sleep 5
```

Comment:
Tried all day to make this work without starting tvh and it doesn't.  I agree it's not very pretty but TVH rewrites the user definition file when it starts up, deliberately wiping all the streaming and dvr settings, even if it's in 4.2 format.   So we have to try to start it here.  Maybe the times it doesn't start will be rare enough that the forum doesn't get flooded.  The fix is just to configure the osmc user manually.  Or else, is there a way to set a flag here and move this code so it gets called from systemd the first time tvh successfully starts after an install/upgrade?